CHANGELOG.md

```diff
@@ -10,12 +10,14 @@
 
 ### Changed
 
+- Bump stackable-operator to 0.87.0 and stackable-versioned to 0.6.0 ([#591]).
```

Comment:
We normally only document user-facing changes in here

Reply:
- The filename is fine, it is a log of changes. Using `CHANGELOG.md` is pretty much the industry standard. The content is aimed at developers.
- Release notes are curated and contain boiled down content from the changelog intended to be read by users.

The thing is: Currently both documents are manually curated by us. The changelog should be automated instead. This is the reason why it feels like they serve the same purpose, but they don't.